frontend/src/components/2.Joining/JoinGroup.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import {Link} from 'react-router-dom';
+import styled from 'styled-components';
+import {useState} from 'react';
+import Help from '../Common/Help';
+import '../Common/Help.css';
+
+const UserInput = styled.input`
+  outline: none;
+  width: 20vw;
+  border-radius: 30px;
+  border: 4px solid #000000;
+  font-size: 4vw;
+  margin: 1vw;
+`;
```

Comment:
I'm generally against in-class styling - would it be possible to extract this out into a stylesheet?

Reply:
From what I have tried at the moment, because we are using the style-components package, it does not like/work to export it out into a css file. What does work is if I make UserInput a Common component which contains the styling inside it.

If this styling does not work being refactored into a css file, we may have to do this, unless we forgo style-component